warehouse/slave.go

```diff
@@ -559,13 +560,12 @@ func processClaimedJob(claimedJob pgnotifier.ClaimT) {
 		return
 	}
 	job.BatchID = claimedJob.BatchID
-	ids, err := processStagingFile(job)
-	job.setStagingFileStatus(err)
```

Comment:
If this function is not used anywhere else, we should remove the function altogether.

Reply:
removed